frontend/src/components/Sidepanel/Panels/Modules/Modules.tsx

```diff
@@ -236,19 +255,22 @@ const Modules = () => {
                     <td onClick={() => handleOpenQuestion(q)}>{`Question ${index + 1}`}</td>
                     <td>
                       <EditButton onClick={() => handleEditQuestion(q)}>Edit</EditButton>
-                      <RemoveButton
+
+                      {currentModule.projects.length > 1 && <RemoveButton
```

Comment:
We still want to show the remove button even if there's only one - I've recently added some styling to disable the button

Reason for this is so the buttons and their locations are familiar, so when the user adds more questions, they already know there will be a remove button
```suggestion
                      <RemoveButton
```